### PR TITLE
add/update item in the cart and mini cart total counter updated

### PR DESCRIPTION
**Pull request by wasil-sayyed**

### CODE REVIEW

File: src/views/Product.vue [hunk 1]
```diff
@@ -187,16 +188,28 @@ export default {
...
 				this.product.quantity = this.selectedQty;
 				this.$store.dispatch("cart/addCartProduct", this.product);
 				this.calloutMsgHandler();
 				this.sizeError = "";
-			} else {
-				this.sizeError = "Please select size";
-			}
+				this.sizeSelected = false;
+			} else {			
+				this.product.size = this.cartData.size;
```
Comment (dinesh-nagrani) [inline src/views/Product.vue:207]: As the logic is same in IF block we can create a generic code for it , instead of repeating it

Comment (wasil-sayyed) [reply src/views/Product.vue:207]: Its not same if executed when new item and size selected and else executed to increase its quantity(if same item is seleccted)

**Comment by dinesh-nagrani** [comment]
this.product.size = this.cartData.size;
this.product.quantity = this.selectedQty;					
this.product.quantity = this.selectedQty;
this.sizeError='' 		
this.sizeSelected = false;

i am talking about this line as these are being repeated in both if and else block , so instead we can create some generic code.
i understand that logic in if and else to add and update the quantity.

File: src/views/Product.vue (hunk 1 above)
Comment (wasil-sayyed) [reply src/views/Product.vue:207]: Done

**Pull request merged**